### PR TITLE
Dev/228: [API] Exam-Service: Update question [PUT] /api/exams/{examId}/problems/{problemId}

### LINKED ISSUE #228: [PUT] /api/exams/{examId}/questions/{questionId}
[PUT] /api/exams/{examId}/questions/{questionId}

Request Body
a question

### PR DESCRIPTION
**Pull request by Johnny850807**
resolve #228

### CODE REVIEW

**Review by Chaoyuuu — APPROVED** [review]
LGTM :)

File: Student-Service/src/main/java/tw/waterball/judgegirl/studentservice/domain/usecases/student/GetStudentUseCase.java [hunk 1]
```diff
@@ -30,19 +32,34 @@ public GetStudentUseCase(StudentRepository studentRepository) {
...
-        return presenter.getStudent();
+        return presenter.getStudents().get(0);
+    }
+
+    public void execute(int studentId, Presenter presenter) {
+        var students = studentRepository.findByIdIn(studentId);
+        if (students.isEmpty()) {
+            throw notFound("student").id(studentId);
+        }
+        presenter.showStudents(students);
```
Comment (Wally5077) [inline Student-Service/src/main/java/tw/waterball/judgegirl/studentservice/domain/usecases/student/GetStudentUseCase.java:46]: var student = studentRepository.findById(studentId).orElseThrow(NotFoundException::new);

File: Student-Service/src/main/java/tw/waterball/judgegirl/studentservice/domain/usecases/student/GetStudentUseCase.java [hunk 2]
```diff
@@ -30,19 +32,34 @@ public GetStudentUseCase(StudentRepository studentRepository) {
...
+        public Integer[] ids;
+
+        public Request(Integer... ids) {
+            this.ids = ids;
+        }
     }
 
     public interface Presenter {
-        void setStudent(Student student);
+        void showStudents(List<Student> students);
```
Comment (Wally5077) [inline Student-Service/src/main/java/tw/waterball/judgegirl/studentservice/domain/usecases/student/GetStudentUseCase.java:63]: void showStudents(Student... students);

**Review by Wally5077 — CHANGES_REQUESTED** [review]
Review done

**Review by Wally5077 — APPROVED** [review]
LGTM

**Pull request merged**